src/uct/api/uct.h

```diff
@@ -904,8 +904,9 @@ struct uct_md_attr {
         size_t               max_alloc; /**< Maximal allocation size */
         size_t               max_reg;   /**< Maximal registration size */
         uint64_t             flags;     /**< UCT_MD_FLAG_xx */
-        uint64_t             reg_mem_types; /** UCS_BIT(uct_memory_type_t) */
-        uct_memory_type_t    mem_type;  /**< Supported(owned) memory type */
+        uint64_t             reg_mem_types; /**< Memory types MD can be registered with */
+        uint64_t             detect_mem_types; /**< Memory types MD can detect with detect_memory_type op */
```

Comment:
This documentation is not very clear either. What is op and what it detects

Reply:
reworded